src/applications/vaos/services/vaos/index.js

```diff
@@ -41,6 +41,18 @@ export function getFacilities(ids, children = false) {
   ).then(parseApiList);
 }
 
+export function getClinicsByLocationAndTypeOfCare(locationId, typeOfCareId) {
```

Comment:
I figured this was easier than a generic `getClinics` call where we had to juggle this two params, plus the clinic id list param that would get used in other circumstances.